app/assets/javascripts/components/data_portal/modals/modal-save-widget.js

```diff
@@ -0,0 +1,109 @@
+(function (App) {
+
+  App.Component.ModalSaveWidget = App.Component.Modal.extend({
+
+    contentTemplate: JST['templates/data_portal/modals/modal-save-widget'],
+
+    defaults: {
+      // See App.Component.Modal for details about this option
+      title: 'Save widget',
+      // See App.Component.Modal for details about this option
+      showTitle: true,
+      // See App.Component.Modal for details about this option
+      allowScroll: true,
+      // See App.Component.Modal for details about this option
+      isAbsolute: true,
+      // See App.Component.Modal for details about this option
+      footer: '<button type="button" class="c-button -medium -white -outline js-button">Add to report</button><div class="group-button"><button type="button" class="c-button -medium -white -disabled js-print">Print</button><button type="button" class="c-button -medium -white -disabled">Download</button><button type="button" data-slide-index="0" class="c-button -medium -white js-slide-button">Share</button></div>',
```

Comment:
The styles of the buttons are wrong: the first one should be white and the other ones with just the outline.